forms-flow-web/src/components/Form/List.js

```diff
@@ -164,6 +173,49 @@ const List = React.memo((props) => {
     uploadFormNode.current?.click();
     return false;
   };
+  const handlePageChange = (type, newState) => {
+    dispatch(setFormSearchLoading(true));
+    let updatedQuery = {query:{...query}};
+    if (type === "sort") {
+      if (isDesigner) {
+        updatedQuery.sort = `${isAscending ? "-" : ""}title`;
+      }else{
+        let updatedSort;
+      if (sortOrder === "asc") {
+        updatedSort = "desc";
```

Comment:
try to Make all Strings as Constant like asc desc

Reply:
resolved